File: app/src/main/java/org/nervos/neuron/plugin/AppTabPlugin.java
package org.nervos.neuron.plugin;

import android.content.Context;
import android.content.Intent;
import android.webkit.JavascriptInterface;

import org.nervos.neuron.activity.AddWebsiteActivity;

public class AppTabPlugin {

    private Context mContext;

    public AppTabPlugin(Context context) {
        mContext = context;
    }

    @JavascriptInterface
    public void startAddWebsitePage() {
        mContext.startActivity(new Intent(mContext, AddWebsiteActivity.class));
    }

}

File: app/src/main/java/org/nervos/neuron/plugin/AppTabPlugin.kt
package org.nervos.neuron.plugin

import android.content.Context
import android.content.Intent
import android.webkit.JavascriptInterface
import org.nervos.neuron.activity.collection.CollectActivity
import org.nervos.neuron.activity.collectwebsite.CollectWebsiteActivity

class AppTabPlugin(private val mContext: Context) {

    @JavascriptInterface
    fun toWebCollection() {
        mContext.startActivity(Intent(mContext, CollectWebsiteActivity::class.java))
    }

    @JavascriptInterface
    fun toErc721() {
        mContext.startActivity(Intent(mContext, CollectActivity::class.java))
    }

}
